### PR TITLE
Update container, Fix replication, port log-level and update readme

### PR DESCRIPTION
**Pull request by Ornias1993**
This is a cleanup, update and fix patch. To ensure all features actually work.
(newsflash: In a lot of cases replication isn't working very well)

As there are replication fixes in v1.4.0 of the container, I updated and tested the new container.
I also patched the bug currently causing many of us not to be able to use replication. The host list for replication NEEDS the **FULL** fqdn. So I added a field to set the cluster name and changed the template to ensure we generate the complete fqdn for creating the replication host list.

I also ported the recently added feature to set the loglevel from upstream, added an actually working installation guide to the readme.

TLDR: This should prevent the FLOOD of replication issues that plaguing this build and are being filed in both the container and helm-chart upstreams

### CODE REVIEW

**Comment by jp-gouin** [comment]
Thanks for your contribution !
I will test it asap.

Can you point to me the reason behind the full fqdn issue ? 
Because in Kubernetes you have DNS résolution based on service name, no need of the cluster.local but maybe it's related to the 1.4 of openldap.

Cheers

**Comment by Ornias1993** [comment]
@jp-gouin I also had issues before 1.4, but the Replication host list, need to be the same as the hostname which is used internally, which resorts to the FQDN. 
It's not the hostname resolution thats at fault (because it does resolve), but it's a strange quirck that has lead to an insane amount of errors on all openldap opstreams, because it's completely weird. But as long as you use fqdn its rock-solid.

With 1.4 it's even more strictly hardcoded to be FQDN though.

**Comment by jp-gouin** [comment]
@Ornias1993 Got it ! 
Thanks!
